app/src/main/AndroidManifest.xml

```diff
@@ -84,6 +84,18 @@
             </intent-filter>
         </service>
 
+        <service
```

Comment:
Not sure how I feel about it, a11y services are sad.
I know you want to preserve the non-system app compat but if we're targeting Lineage I wouldn't want it.